src/views/ScorePage/ScorePage.js

```diff
@@ -1,33 +1,34 @@
 import './ScorePage.css';
-import { scoreLocalStorage, saveHighScore } from '../../score/user-score';
 import Button from '../../components/Button/Button';
 import '../../components/Button/Button.css';
 
+import { QuizSettings } from '../quiz-settings/quiz-settings';
+import { Answers } from '../QuizView/quizView';
+import Answer from '../../components/Answer/Answer';
+
 export function renderScorePage() {
-  const highScores = JSON.parse(localStorage.getItem('highScores')) || [];
-  const recentUserScore = 12;
-  const questionsNumber = 12;
+  const recentUserScore = Answers.reduce((a, b) => a + b.getScore(), 0);
```

Comment:
This is a short version but it would be easier to read with parameters:  'sum' (or 'acc'), 'ans' (or 'answer') as parameters.